modules/dxp/apps/digital-sales-room/digital-sales-room-web/src/main/resources/META-INF/resources/js/components/DSRTypes.tsx

```diff
@@ -30,11 +30,16 @@ export type TDSRDataContext = {
 		primaryColor?: null | string;
 		roomName?: null | string;
 		secondaryColor?: null | string;
+		share?: null | string;
```

Comment:
why do you keep these nulls + optionals? 🥹

Reply:
to manage the optional key or the key with a null value